src/main/resources/config/application-prod.yml

```diff
@@ -13,10 +13,13 @@
 # http://docs.spring.io/spring-boot/docs/current/reference/html/common-application-properties.html
 # ===================================================================
 
+logging:
+    level:
+        ROOT: INFO
+        com.icthh.xm.uaa: INFO
+        io.github.jhipster: INFO
+
 spring:
-    profiles:
-        active: prod
-        include: swagger
```

Comment:
Suggest to check if removing swagger profile will not prevent from exposing swagger at all.

Reply:
returned back